pelux.xml

```diff
@@ -71,7 +71,7 @@
 
   <!-- Qt + Qt Automotive support stuff -->
   <project remote="code.qt"
-           upstream="5.12"
+           upstream="5.13"
            revision="refs/tags/v5.13.0"
```

Comment:
Approved if revision is updated to current HEAD of 5.13, i.e. 6699712b98754bb7f3199589374b8db2da8517ce.